developer-docs-site/docs/nodes/validator-node/operator/node-requirements.md

```diff
@@ -38,6 +38,8 @@ For running an Aptos **validator node and validator fullnode** we recommend the
 - **GCP**
     - n2-standard-16 (if use local SSD)
     - n2-standard-32 + pd-ssd with 40K IOPS.
+- **CherryServers**
+    - Intel Gold 5315Y (8c/16t - 3.2GHz, 32GB RAM, 2TB NVMe)
```

Comment:
I think the issue of adding this and this alone is that it is ignoring the rest of the requirements in terms of firewalls. It is a somewhat obscure mention with no mention in this document otherwise. Perhaps you'd like to contribute a complete tutorial on the topic?

Reply:
Hey,
By default, all the ports are open on CherryServers. 
UFW can ensure that only the required ports are open ( CherryServers have a guide here: https://www.cherryservers.com/blog/how-to-configure-ubuntu-firewall-with-ufw). But this is relatively standard for all the providers, so I am not sure this is relevant.

Reply:
Can we leave it as it is, or should I add more info under the "local SSD vs. network storage" and "ports"?

Reply:
I think we need a bare metal (cloud) tutorial and then we can have a list of verified vendors.